docs/RECIPES.md

```diff
@@ -137,3 +137,34 @@ EOF
 ```shell
 oc apply -f preflight.yaml
 ```
+
+## Container Policy
+These examples are shown using the Container policy against a container image
+(e.g. `preflight check container <image>`). Container policy only runs as a binary on your workstation. Check the latest
+[release](https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases) for the binary that matches your operating system.
+
+You will also need:
+- Your container image published to a container registry
+- A Certification Project ID of the project that was set up in Red Hat Partner Connect
```

Comment:
add something parenthetical along these lines: 

You can find the project ID embedded in the URL of the Overview page: https://connect.redhat.com/projects/<Project ID>/overview. Do NOT use the field labelled with "PID:" on the Overview page proper if it contains dashes - these PID values will be different from the Project ID embedded in the URL.

Reply:
Is there not a better place to get the information than the URL? If not, we need to change that.

Reply:
@jfrancin The value in the PID field is the _same_ as the value in the URL, the only difference is the leading `ospid` which we are stripping off if the user gives it to us.

Reply:
_Not Always._ For any container project that was created prior to the mid-july 2021 Cutover date from the OpenShift 3.11-based container registry to the OpenShift 4.8 based registry, the project ID in the URL and the PID in the Overview page _will be different_. Here's an example from one of IBM's thousands of container projects:

Project: IBM Sterling Order Management Base Image - Professional Edition, updated last night:
URL: https://connect.redhat.com/projects/5e7a5fc31c86a3163d1a6997/overview
PID on the Overview page: ospid-a1f6607d-1247-4199-a6b4-da5f48ffb77c

So you can only use the OSPID if and only if it matches the project ID in the URL. If it doesn't, then the URL is correct and the OSPID is wrong.

Observationally, I also note that projects that were created prior to the July cutover have Pyxis project IDs that lead with a '5', while all projects created post the cutover have project IDs that lead with a '6'. (You might want to check with Alex Misstear to see if that was an intentional bump of the project IDs or not; if it was, you can use that as an indicator that the OSPID will not match the project ID.)